### PR TITLE
[MS] Update input visual issues (password error and outline on focus)

### LINKED ISSUE #6149: [🐛 Bug] Input outline not visible on some modals
See exemple on modal: Create New Workspace
Actual fix: A padding has been added around the item to add space for the outline

Image

### LINKED ISSUE #6460: Replace toast modal error to helperText on `LoginPage`

### PR DESCRIPTION
**Pull request by fabienSvtr**
## Describe your changes

- Outline is visible without adding padding
image

- Toast error message changed to input helper text
image

## Checklist

Before you submit this pull request, please make sure to:

- [x] Keep changes in the pull request as small as possible
   
- [x] Ensure the commit history is sanitized
   
- [x] Give a meaningful title to your PR
- [x] Describe your changes
   
- [x] Link any related issue in the description
   
- [x] Link any dependent pull request in the description

closes #6149, closes #6460

### CODE REVIEW

File: client/src/components/core/ms-input/MsPasswordInput.vue [hunk 1]
```diff
@@ -54,6 +61,8 @@ const hasFocus = ref(false);
 const props = defineProps<{
   label: string;
   modelValue?: string;
+  isWrong?: boolean;
```
Comment (Max-7) [inline client/src/components/core/ms-input/MsPasswordInput.vue:64]: You don't need two variables. You can just rely on wether the errorMessage is set or not.

File: client/src/components/core/ms-input/MsPasswordInput.vue [hunk 2]
```diff
@@ -1,48 +1,55 @@
...
+      <ion-input
+        class="form-input input"
+        ref="inputRef"
+        aria-labelledby="passwordLabel"
+        :type="passwordVisible ? 'text' : 'password'"
+        @ion-input="onChange($event.target.value)"
+        :value="modelValue"
+        id="ms-password-input"
+        :clear-on-edit="false"
+        @keyup.enter="onEnterPress()"
```
Comment (Max-7) [inline client/src/components/core/ms-input/MsPasswordInput.vue:24]: You've removed `@ion-focus` and `@ion-blur`. Probably a bad conflict resolution, but be careful.

File: client/src/locales/fr-FR.json [hunk 3]
```diff
@@ -106,6 +106,7 @@
             "forgottenPassword": "Mot de passe oublié ?",
             "emailLabel": "E-mail",
             "passwordLabel": "Mot de passe",
+            "passwordError": "Le mot de passe est incorrect",
```
Comment (Max-7) [inline client/src/locales/fr-FR.json:109]: English text ends with a `.`, French text does not.

File: client/src/views/home/LoginPage.vue [hunk 4]
```diff
@@ -56,15 +58,18 @@
...
 import OrganizationCard from '@/components/organizations/OrganizationCard.vue';
-import { AvailableDevice } from '@/parsec';
+import { AvailableDevice, ClientStartError } from '@/parsec';
 import { IonButton, IonCard, IonCardContent, IonCardHeader, IonFooter, IonIcon, IonTitle } from '@ionic/vue';
 import { logIn } from 'ionicons/icons';
 import { onMounted, ref } from 'vue';
 
 const props = defineProps<{
   device: AvailableDevice;
+  validator?: IValidator;
```
Comment (Max-7) [inline client/src/views/home/LoginPage.vue:71]: `validator` is not used.

File: client/src/views/home/LoginPage.vue [hunk 5]
```diff
@@ -26,6 +26,8 @@
             v-model="password"
             @on-enter-keyup="onLoginClick()"
             id="ms-password-input"
+            :is-wrong="props.error ? true : false"
+            :error-message="$t('HomePage.organizationLogin.passwordError')"
```
Comment (Max-7) [inline client/src/views/home/LoginPage.vue:30]: Set `error-message` only if `props.error` is set, you don't need `is-wrong`.

**Review by Max-7 — CHANGES_REQUESTED** [review]

File: client/src/locales/fr-FR.json (hunk 3 above)
Comment (fabienSvtr) [reply client/src/locales/fr-FR.json:109]: Done

File: client/src/views/home/LoginPage.vue (hunk 4 above)
Comment (fabienSvtr) [reply client/src/views/home/LoginPage.vue:71]: Removed.

File: client/src/views/home/LoginPage.vue (hunk 5 above)
Comment (fabienSvtr) [reply client/src/views/home/LoginPage.vue:30]: Done

File: client/src/components/core/ms-input/MsPasswordInput.vue (hunk 1 above)
Comment (fabienSvtr) [reply client/src/components/core/ms-input/MsPasswordInput.vue:64]: Done

File: client/src/components/core/ms-input/MsPasswordInput.vue (hunk 2 above)
Comment (fabienSvtr) [reply client/src/components/core/ms-input/MsPasswordInput.vue:24]: Yes, because of rebase I think. I added it back.

**Review by Max-7 — APPROVED** [review]

**Pull request merged**